runtimevar/constantvar/constantvar.go

```diff
@@ -24,13 +38,51 @@ package constantvar // import "gocloud.dev/runtimevar/constantvar"
 import (
 	"context"
 	"errors"
+	"fmt"
+	"net/url"
 	"time"
 
 	"gocloud.dev/gcerrors"
 	"gocloud.dev/runtimevar"
 	"gocloud.dev/runtimevar/driver"
 )
 
+func init() {
+	runtimevar.DefaultURLMux().RegisterVariable(Scheme, &urlOpener{})
+}
+
+// Scheme is the URL scheme constantvar registers its URLOpener under on blob.DefaultMux.
+const Scheme = "constant"
+
+type urlOpener struct{}
```

Comment:
This should be exported so that Wire users can use the implementation.

Reply:
Done.